dev/tests/cert.rs

```diff
@@ -5,7 +5,9 @@ use dev::{
     timeout::{spawn_timeout, timeout},
 };
 use dis_spawner::{messages::cert::SetAcmeDnsRecord, nats::TypedNats, types::ClusterName};
-use dis_spawner_drone::{cert::acme::AcmeEabConfiguration, cert::CertOptions, keys::KeyCertPathPair};
+use dis_spawner_drone::{
+    cert::acme::AcmeEabConfiguration, cert::CertOptions, keys::KeyCertPathPair,
+};
```

Comment:
I think I'm the one with a misconfigured rustfmt probably, but might be a good idea to put a rustfmt toml in the root dir.

Reply:
What config are you using? I am just using defaults but if you are opinionated about the style I'm open to different settings.